README.md

```diff
@@ -1,61 +1,43 @@
 # rescript-webapi
 
+[![npm](https://img.shields.io/npm/v/rescript-webapi.svg)](https://npmjs.org/rescript-webapi)
+
 Bindings to the DOM and other browser-specific web APIs.
 
 ## Stability
 
-Until the library reaches 1.0, expect major breaking changes to the API. See the [1.0 project](https://github.com/tinymce/rescript-webapi/projects/1) for the current progress towards that goal.
-
-## TODO
-
-everything.
-
-## License and Credits
-
-All code is licensed as MIT. See [LICENSE](LICENSE).
-
-This project has been forked from [bs-webapi](https://github.com/reasonml-community/bs-webapi-incubator) after it was mostly abandoned. Rather than offer to continue that project, we took the opportunity to make major changes to the API and bring it up to modern ReScript development practices by renaming it.
-
-Primary development on this project is sponsored by [Tiny](https://github.com/tinymce/).
-
-
-# NOTE
-This project is in the process of being forked; what follows is the original readme.
-
-# bs-webapi
-
-Experimental bindings to the DOM and other Web APIs.
-
-[![npm](https://img.shields.io/npm/v/bs-webapi.svg)](https://npmjs.org/bs-webapi)
-[![Travis](https://img.shields.io/travis/reasonml-community/bs-webapi-incubator/master.svg)](https://travis-ci.org/reasonml-community/bs-webapi-incubator)
-
-API docs are available at
-https://reasonml-community.github.io/bs-webapi-incubator/api/Webapi/ , but
-documentation comments are sparse as the code mostly just consists of
-external declarations with type signatures. The bindings generally also
-correspond very well to the Web APIs they bind to, so using MDN along
-with GitHub should go a long way.
+This project is in the process of being forked; until the library reaches 1.0, expect major breaking changes to the API. See the [1.0 project](https://github.com/tinymce/rescript-webapi/projects/1) for the current progress towards that goal.
 
 ## Installation
 
 ```
-npm install bs-webapi
+npm install rescript-webapi
 ```
-Then add `bs-webapi` to `bs-dependencies` in your `bsconfig.json`. A minimal example:
+
+Then add `rescript-webapi` to `bs-dependencies` in your `bsconfig.json`. A minimal example:
+
 ```
 {
   "name": "my-thing",
   "sources": "src",
-  "bs-dependencies": ["bs-webapi"]
+  "bs-dependencies": ["rescript-webapi"]
 }
 ```
 
 ## Usage
 
-See the [examples folder](https://github.com/reasonml-community/bs-webapi-incubator/blob/master/examples/)
+See the [examples folder](https://github.com/tinymce/rescript-webapi/tree/main/examples).
 
 Please only use the modules exposed through the toplevel module `Webapi`, for example `Webapi.Dom.Element`. In particular, don't use the 'flat' modules like `Webapi__Dom__Element` as these are considered private and are not guaranteed to be backwards-compatible.
 
+## License and Credits
```

Comment:
I've moved this section below installation and usage as I feel like those two sections are more important - I hope this is ok!

Reply:
I prefer to put it at the very end; it's only near the top right now because I had kept the original readme intact.